ci/Jenkinsfiles/export.groovy

```diff
@@ -0,0 +1,329 @@
+  }
+
+  stages {
+
+    stage('Check Parameters') {
+      steps {
+        script {
+          echo """
+          ----------------------------------------
+          Nuxeo version:                 '${params.NUXEO_VERSION}'
+          Explorer package:              '${NUXEO_EXPLORER_PACKAGE}'
+
+          Connect Explorer URL:          '${CONNECT_EXPLORER_URL}'
+          Connect Export URL:            '${CONNECT_EXPORT_URL}'
+          Download Packages from Prod:   '${params.DOWNLOAD_PACKAGES_FROM_PROD}'
+          Package List:                  '${EXPORT_PACKAGE_LIST}'
+          Snapshot Name:                 '${params.SNAPSHOT_NAME}'
+          Perform Json Export:           '${params.PERFORM_JSON_EXPORT}'
+
+          Upload Export:                 '${UPLOAD_EXPORT}'
+          Upload URL:                    '${UPLOAD_URL}'
+          Upload as Promoted:            '${params.UPLOAD_AS_PROMOTED}'
+          ----------------------------------------
+          """
+          if (params.NUXEO_VERSION.isEmpty()) {
+            currentBuild.result = 'ABORTED';
+            def message = 'NUXEO_VERSION parameters is mandatory'
+            currentBuild.description = "${message}"
+            echo "${message}"
+            error(currentBuild.description)
+          }
+        }
+      }
+    }
+
+    stage('Set Labels') {
+      steps {
+        container('maven') {
+          echo """
+          ----------------------------------------
+          Set Kubernetes resource labels
+          ----------------------------------------
+          """
+          echo "Set label 'branch: export' on pod ${NODE_NAME}"
+          sh """
+            kubectl label pods ${NODE_NAME} branch=export
+          """
+        }
+      }
+    }
+
+    stage('Build Docker Image') {
+      steps {
+        container('maven') {
+          echo """
+          ----------------------------------------
+          Build Explorer Export Docker Image
+          ----------------------------------------
+          Image tag: ${VERSION}
+          Registry: ${DOCKER_REGISTRY}
+          """
+          script {
+            def moduleDir = 'docker/nuxeo-explorer-export-docker'
+            def exportCredsId = "${params.DOWNLOAD_PACKAGES_FROM_PROD ? 'explorer-connect-prod-clid' : 'explorer-connect-preprod-clid'}"
+            // push images to the Jenkins X internal Docker registry
+            withCredentials([string(credentialsId: exportCredsId, variable: 'CONNECT_EXPORT_CLID')]) {
+              sh """
+                envsubst < ${moduleDir}/skaffold.yaml > ${moduleDir}/skaffold.yaml~gen
+                skaffold build -f ${moduleDir}/skaffold.yaml~gen
+                # waiting skaffold + kaniko + container-stucture-tests issue
+                #  see https://github.com/GoogleContainerTools/skaffold/issues/3907
+                docker pull ${DOCKER_REGISTRY}/nuxeo/nuxeo-explorer-export:${VERSION}
```

Comment:
Not needed as there's no container-structure-tests, the two comments above neither.